modules/js/src/core_bindings.cpp

```diff
@@ -586,5 +598,7 @@ EMSCRIPTEN_BINDINGS(binding_utils)
         .function("Shutdown", &ORB_SLAM2::System::Shutdown)
         .function("GetTrackingState", &ORB_SLAM2::System::GetTrackingState)
         .function("GetTrackedMapPoints", &ORB_SLAM2::System::GetTrackedMapPoints)
-        .function("GetTrackedKeyPointsUn", &ORB_SLAM2::System::GetTrackedKeyPointsUn);
+        .function("GetTrackedKeyPointsUn", &ORB_SLAM2::System::GetTrackedKeyPointsUn)
+        .function("GetKeyFramesInMap", &ORB_SLAM2::System::GetKeyFramesInMap)
+        .function("GetMapPointsInMap", &ORB_SLAM2::System::GetMapPointsInMap);
```

Comment:
Ditto.

Reply:
They are used to print the numbers of frames and points in console.

Reply:
Can we get the same info from API for actual key frames or map points?

Reply:
I think we can delete this after implementing the UI.

Reply:
OK. Let's address this in following PR.